syntax = "proto3";

package xgrpc;

message RpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_length = 3;
}
